start

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+npm start
```

Comment:
Is this file required for docker?

Reply:
not required but added for future reference if we need to run more then one npm command.